### PR TITLE
Update for play 2.6.x

### PR DESCRIPTION
**Pull request by kijuky**
# Summary

Update Play 2.6.x

# Detail

- fix readme
- fix Build.scala
- fix test
- fix sample

| library | before | after | remarks |
|---|---|---|---|
| scala | 2.11.8 | 2.11.12 | |
| sbt | 0.13.11 | 0.13.15 | |
| play | 2.5.19 | 2.6.25 | |
| scalatest | 2.2.3 | 3.2.15 | |
| sbt-scoverage | 1.0.2 | 1.6.1 | |

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,15 +7,17 @@ jobs:
...
-          "adopt@1.8.0-292"
+          8
         ]
     steps:
       - name: Checkout
         uses: actions/checkout@v3
       - name: Setup Scala
-        uses: olafurpg/setup-scala@v13
+        uses: actions/setup-java@v3
         with:
```
Comment (lloydmeta) [inline .github/workflows/test.yml]: Is this change needed for this PR? seems like GH does't like to run action changes in the same PR, so if possible, it would be good procedurally to have these in a separate one.

Comment (kijuky) [reply .github/workflows/test.yml]: OK. I will remove this change from this PR. And make this change another PR #7 .

**Review by lloydmeta — APPROVED** [review]
LGTM.

**Comment by lloydmeta** [comment]
Thanks for this. Just pushed 1.4 with these changes to maven central.

**Comment by kijuky** [comment]
Thank you for your work! It helped me a lot.
